client/components/course/Revisions/EntitySidebar.vue

```diff
@@ -56,6 +56,11 @@ export default {
 <style lang="scss" scoped>
 $revision-padding: 32px;
 
+@mixin selected-revision() {
```

Comment:
Parens are redundant when you have no args. Also on `@include` statements ⬇️  as well.